doku/src/printers/toml/formatting/layout.rs

```diff
@@ -0,0 +1,151 @@
+use crate::*;
+
+/// Determines whether the document should contain one or two columns.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub enum Layout {
+    /// Prints types and comments inline, in a single chunk of text:
+    ///
+    /// ```
+    /// use doku::Document;
+    ///
+    /// #[derive(Document)]
+    /// struct Person {
+    ///     /// First name
+    ///     /// (aka forename)
+    ///     fname: String,
+    ///
+    ///     /// Last name
+    ///     /// (aka surname)
+    ///     lname: String,
+    /// }
+    ///
+    /// let fmt = doku::toml::Formatting {
+    ///     layout: doku::toml::Layout::OneColumn,
+    ///     ..Default::default()
+    /// };
+    ///
+    /// let doc = doku::to_toml_fmt::<Person>(&fmt);
+    ///
+    /// doku::assert_doc!(r#"
+    ///   ## First name
+    ///   ## (aka forename)
+    ///   fname = "string"
+    ///   ## Last name
+    ///   ## (aka surname)
+    ///   lname = "string"
+    /// "#, doc);
+    /// ```
+    OneColumn,
+
+    /// Prints types and comments in two separate columns.
+    ///
+    /// ```
+    /// use doku::Document;
+    ///
+    /// #[derive(Document)]
+    /// struct Person {
+    ///     /// First name
+    ///     /// (aka forename)
+    ///     fname: String,
+    ///
+    ///     /// Last name
+    ///     /// (aka surname)
+    ///     lname: String,
+    /// }
+    ///
+    /// let fmt = doku::toml::Formatting {
+    ///     layout: doku::toml::Layout::TwoColumns {
+    ///         align: true,
+    ///         spacing: 1,
+    ///     },
+    ///     ..Default::default()
+    /// };
+    ///
+    /// let doc = doku::to_toml_fmt::<Person>(&fmt);
+    ///
+    /// doku::assert_doc!(r#"
+    ///   fname = "string" # First name
+    ///                    ## (aka forename)
+    ///   lname = "string" # Last name
+    ///                    ## (aka surname)
+    /// "#, doc);
+    /// ```
+    TwoColumns {
+        /// Whether the two columns should be aligned or not:
+        ///
+        /// ```
+        /// use doku::Document;
+        ///
+        /// #[derive(Document)]
+        /// struct Person {
+        ///     /// First name
+        ///     /// (aka forename)
+        ///     fname: String,
+        ///
+        ///     /// Last name
+        ///     /// (aka surname)
+        ///     lname: String,
+        /// }
+        ///
+        /// let fmt = doku::toml::Formatting {
+        ///     layout: doku::toml::Layout::TwoColumns {
+        ///         align: false,
+        ///         spacing: 1,
+        ///     },
+        ///     ..Default::default()
+        /// };
+        ///
+        /// let doc = doku::to_toml_fmt::<Person>(&fmt);
+        ///
+        /// doku::assert_doc!(r#"
+        ///   fname = "string" # First name
```

Comment:
Since both `fname` and `lname` have the same length, it's not possible to see that this layout actually aligns the left and right colum (it's different for JSON, where there's a comma in `fname: ...,` making it one character longer).

Maybe we could replace `fname` / `lname` to just `first_name` and `last_name` (both here and in the JSON example)? That'd make the alignment pretty obvious.